### PR TITLE
LPS-197944 [PRM] Freemark syntax error message appears after saving the MDF as draft

### PR DESCRIPTION
**Pull request by Manda-vit**

### CODE REVIEW

**Comment by Manda-vit** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/cancel-button-mdf-claim/index.html [hunk 1]
```diff
@@ -3,9 +3,9 @@
 [#if mdfClaimId?has_content]
 	[#assign mdfClaim = restClient.get("/c/mdfclaims/${mdfClaimId}") /]
 	[#assign permissionAction = restClient.get("/c/permissionactions?filter=object%20eq%20%27MDF%20Claim%27&pageSize=-1") /]
-	[#assign hasCancelAction = permissionAction.items?filter(permissionAction -> permissionAction.action == "CANCEL")?has_content /]
+	[#assign hasCancelAction = permissionAction.items?filter(permissionAction -> (permissionAction.action)! == "CANCEL")?has_content /]
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/cancel-button-mdf-claim/index.html:6]:
```suggestion
	[#assign hasCancelAction = (permissionAction.items?filter(permissionAction -> permissionAction.action == "CANCEL")?has_content)! /]
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html [hunk 2]
```diff
@@ -47,19 +47,19 @@ <h6 class="text-neutral-10">
 					</div>
 				[/#if]
 
-				[#if activity.telemarketingMetrics?has_content]
+				[#if activity.telemarketingMetrics?has_content && (activity.telemarketingMetrics?boolean)! != false]
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html:50]:
```suggestion
				[#if (activity.telemarketingMetrics?has_content)! && (activity.telemarketingMetrics?boolean)! != false]
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html [hunk 3]
```diff
@@ -47,19 +47,19 @@ <h6 class="text-neutral-10">
...
 
 						<div class="mx-2 text-neutral-10">
-							${activity.telemarketingMetrics}
+							${(activity.telemarketingMetrics)!}
 						</div>
 					</div>
 				[/#if]
 
-				[#if activity.telemarketingScript?has_content]
+				[#if activity.telemarketingScript?has_content && (activity.telemarketingScript?boolean)! != false]
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html:62]:
```suggestion
				[#if (activity.telemarketingScript?has_content)! && (activity.telemarketingScript?boolean)! != false]
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Comment by Manda-vit** [comment]
ci:test:stable

**Comment by Manda-vit** [comment]
ci:test:stable

**Comment by Manda-vit** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html [hunk 4]
```diff
@@ -292,7 +292,7 @@ <h6 class="mb-0 mt-4 text-neutral-10">
 [#if mdf_claim_id?has_content]
 	[#assign activities = restClient.get("/c/mdfclaims/${mdf_claim_id}/mdfClmToMDFClmActs?nestedFields=mdfClmActToMDFClmBgts,mdfClmActToMDFActDocs").items /]
 
-	[#list activities?filter(activity -> activity.selected == true) as activity]
+	[#list (activities?filter(activity -> activity.selected)! == true) as activity]
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html:295]:
```suggestion
	[#list (activities?filter(activity -> activity.selected))! == true) as activity]
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.html [hunk 5]
```diff
@@ -124,7 +124,7 @@
 [/#macro]
 
 [#macro leads_table activity]
-	[#assign items = [{"title": "Is a lead list an outcome of this activity?", "value": activity.leadGenerated?string("Yes", "No")!"-" }, { "title": "Target # of Leads", "value": (activity.targetOfLeads)!"-" }, { "title": "Lead Follow Up strategy", "value": (activity.leadFollowUpStrategies)!"-" }, { "title": "Details on Lead Follow Up", "value": (activity.detailsLeadFollowUp)!"-" }] /]
+	[#assign items = [{"title": "Is a lead list an outcome of this activity?", "value": activity.leadGenerated?string("Yes", "No") }, { "title": "Target # of Leads", "value": (activity.targetOfLeads)!"-" }, { "title": "Lead Follow Up strategy", "value": (activity.leadFollowUpStrategies)!"-" }, { "title": "Details on Lead Follow Up", "value": (activity.detailsLeadFollowUp)!"-" }] /]
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.html:127]:
```suggestion
	[#assign items = [{"title": "Is a lead list an outcome of this activity?", "value": (activity.leadGenerated?string("Yes", "No"))! }, { "title": "Target # of Leads", "value": (activity.targetOfLeads)!"-" }, { "title": "Lead Follow Up strategy", "value": (activity.leadFollowUpStrategies)!"-" }, { "title": "Details on Lead Follow Up", "value": (activity.detailsLeadFollowUp)!"-" }] /]
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.html [hunk 6]
```diff
@@ -134,7 +134,7 @@
 [/#macro]
 
 [#macro budgets_table budgets currency_key="USD"]
-	[#assign items = budgets?map(budget -> {"title": (budget.expense.name)!"-", "value": (budget.cost?number?string["¤,##0.00;; currencyCode=${currency_key}"])!"-"}) /]
+	[#assign items = budgets?map(budget -> {"title": (budget.expense.name)!"-", "value": budget.cost?number?string["¤,##0.00;; currencyCode=${currency_key}"]}) /]
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.html:137]:
```suggestion
	[#assign items = (budgets?map(budget -> {"title": (budget.expense.name)!"-", "value": (budget.cost?number?string["¤,##0.00;; currencyCode=${currency_key}"])!}))! /]
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html (hunk 4 above)
Comment (Manda-vit) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html:295]: I tried this way:

[#list ((activities?filter(activity -> activity.selected))! == true) as activity]

**Comment by Manda-vit** [comment]
ci:test:sf

**Comment by Manda-vit** [comment]
ci:test:stable

**Review by jairmedeiros — APPROVED** [review]

**Comment by Manda-vit** [comment]
ci:test:relevant
ci:test:stable

**Comment by Manda-vit** [comment]
ci:test:stable

**Comment by Manda-vit** [comment]
ci:test:sf

**Comment by Manda-vit** [comment]
ci:test:sf

**Comment by Manda-vit** [comment]
ci:test:stable

**Comment by Manda-vit** [comment]
ci:test:sf

**Comment by Manda-vit** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html [hunk 7]
```diff
@@ -47,19 +47,19 @@ <h6 class="text-neutral-10">
 					</div>
 				[/#if]
 
-				[#if activity.telemarketingMetrics?has_content]
+				[#if (activity.telemarketingMetrics?has_content)! != false]
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html:50]:
```suggestion
				[#if (activity.telemarketingMetrics?has_content)!false != false]
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html [hunk 8]
```diff
@@ -115,11 +115,11 @@ <h6 class="mb-0 mt-4 text-neutral-10">
 						</thead>
 
 						<tbody>
-							[#list activity.mdfClmActToMDFClmBgts?filter(budget -> budget.selected == true) as budget]
+							[#list (activity.mdfClmActToMDFClmBgts?filter(budget -> budget.selected != false))![] as budget]
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html:118]:
```suggestion
							[#list (activity.mdfClmActToMDFClmBgts?filter(budget -> budget.selected == true))![] as budget]
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html [hunk 9]
```diff
@@ -47,19 +47,19 @@ <h6 class="text-neutral-10">
...
 
 						<div class="mx-2 text-neutral-10">
-							${activity.telemarketingMetrics}
+							${(activity.telemarketingMetrics)!}
 						</div>
 					</div>
 				[/#if]
 
-				[#if activity.telemarketingScript?has_content]
+				[#if (activity.telemarketingScript?has_content)!false && (activity.telemarketingScript?boolean)!false != false]
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html:62]:
```suggestion
				[#if (activity.telemarketingScript?has_content)!false]
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html [hunk 10]
```diff
@@ -77,7 +77,7 @@ <h6 class="text-neutral-10">Telemarketing Script</h6>
 					</div>
 				[/#if]
 
-				[#if activity.videoLink?has_content]
+				[#if (activity.videoLink?has_content)!false && (activity.videoLink?boolean)!false != false]
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html:80]:
```suggestion
				[#if (activity.videoLink?has_content)!false]
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html [hunk 11]
```diff
@@ -89,14 +89,14 @@ <h6 class="text-neutral-10">
 					</div>
 				[/#if]
 
-				[#if activity.metrics?has_content]
+				[#if (activity.metrics?has_content)!false && (activity.metrics?boolean)!false != false]
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html:92]:
```suggestion
				[#if (activity.metrics?has_content)!false]
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html [hunk 12]
```diff
@@ -128,7 +128,7 @@ <h6 class="mb-0 mt-4 text-neutral-10">
 										</div>
 									</td>
 									<td class="border-0">
-										[#if budget.invoice?has_content]
+										[#if (budget.invoice?has_content)!false && (budget.invoice?boolean)!false != false]
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html:131]:
```suggestion
										[#if (budget.invoice?has_content)!false]
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html [hunk 13]
```diff
@@ -159,14 +159,14 @@ <h6 class="mb-0 mt-4 text-neutral-10">
...
-							[#if activity.typeActivity.key == "prmtact001"]
+							[#if (activity.typeActivity.key)! == "prmtact001"]
 								<tr>
 									<td class="border-0">
 										<div class="font-weight-bold mb-3 text-neutral-10">
 											Event Program
 										</div>
 
-										[#if activity.eventProgram?has_content]
+										[#if (activity.eventProgram?has_content)!false && (activity.eventProgram?boolean)!false != false]
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html:169]:
```suggestion
										[#if (activity.eventProgram?has_content)!false]
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html [hunk 14]
```diff
@@ -204,7 +204,7 @@ <h6 class="mb-0 mt-4 text-neutral-10">
 											Event Photos
 										</div>
 
-										[#list activity.mdfClmActToMDFActDocs?filter(document -> document.proofOfPerformanceType.key == "eventPhotos") as document]
+										[#list ((activity.mdfClmActToMDFActDocs?filter(document -> document.proofOfPerformanceType.key) == "eventPhotos"))![] as document]
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html:207]:
```suggestion
										[#list (activity.mdfClmActToMDFActDocs?filter(document -> document.proofOfPerformanceType.key == "eventPhotos"))![] as document]
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html [hunk 15]
```diff
@@ -223,7 +223,7 @@ <h6 class="mb-0 mt-4 text-neutral-10">
 											Event Collaterals
 										</div>
 
-										[#list activity.mdfClmActToMDFActDocs?filter(document -> document.proofOfPerformanceType.key == "eventCollaterals") as document]
+										[#list ((activity.mdfClmActToMDFActDocs?filter(document -> document.proofOfPerformanceType.key) == "eventCollaterals"))![] as document]
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html:226]:
```suggestion
										[#list (activity.mdfClmActToMDFActDocs?filter(document -> document.proofOfPerformanceType.key == "eventCollaterals"))![] as document]
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html [hunk 16]
```diff
@@ -290,7 +290,7 @@ <h6 class="mb-0 mt-4 text-neutral-10">
 [/#macro]
 
 [#if mdf_claim_id?has_content]
-	[#assign activities = restClient.get("/c/mdfclaims/${mdf_claim_id}/mdfClmToMDFClmActs?nestedFields=mdfClmActToMDFClmBgts,mdfClmActToMDFActDocs").items /]
+	[#assign activities = (restClient.get("/c/mdfclaims/${mdf_claim_id}/mdfClmToMDFClmActs?nestedFields=mdfClmActToMDFClmBgts,mdfClmActToMDFActDocs").items)! /]
 
 	[#list activities?filter(activity -> activity.selected == true) as activity]
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-claim-detail-collection/fragments/claim-detail-activity-list/index.html:295]:
```suggestion
	[#list (activities?filter(activity -> activity.selected == true))![] as activity]
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.html [hunk 17]
```diff
@@ -26,32 +26,32 @@
...
-	[#if activity.tactic.name != "Email Campaign"]
+	[#if (activity.tactic.name)! != "Email Campaign"]
 		[#assign digital_marketing_fields = digital_marketing_fields + [ { "title": "Specific sites to be used", "value": (activity.specificSites)!"-" }, { "title": "Keywords for PPC campaigns", "value": (activity.keywordsForPPCCampaigns)!"-" }, { "title": "Ad (any size/type)", "value": (activity.ad)!"-" }] /]
-	[#elseif activity.tactic.name = "Email Campaign"]
+	[#elseif (activity.tactic.name)! = "Email Campaign"]
 		[#assign digital_marketing_fields = digital_marketing_fields + [{"title": "Landing page copy", "value": (activity.landingPageCopy)!"-" }] /]
 	[/#if]
 
-	[#if activity.nurtureDripCampaign?has_content]
+	[#if (activity.nurtureDripCampaign?has_content)! && (activity.nurutureDripCampaing?boolean)! =! false]
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.html:37]:
```suggestion
	[#if (activity.nurtureDripCampaign?has_content)!false]
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.html [hunk 18]
```diff
@@ -26,32 +26,32 @@
...
 		[#assign digital_marketing_fields = digital_marketing_fields + [{"title": "Nurture or drip campaign?", "value": activity.nurtureDripCampaign?boolean?string("Yes", "No")!"-"}] /]
 	[/#if]
 
-	[#if activity.manySeries?has_content]
-		[#assign digital_marketing_fields = digital_marketing_fields + [{"title": "How many in series?", "value": (activity.manySeries)!"-"}] /]
+	[#if (activity.manySeries?has_content)! && (activity.manySeries?boolean)! != false]
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.html:41]:
```suggestion
	[#if (activity.manySeries?has_content)!false]
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.html [hunk 19]
```diff
@@ -26,32 +26,32 @@
...
+		[#assign digital_marketing_fields = digital_marketing_fields + [{"title": "How many in series", "value": activity.manySeries}] /]
 	[/#if]
 
-	[#if activity.assetsLiferayRequired?has_content]
+	[#if (activity.assetsLiferayRequired?has_content)!]
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.html:45]:
```suggestion
	[#if (activity.assetsLiferayRequired?has_content)!false]
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.html [hunk 20]
```diff
@@ -26,32 +26,32 @@
...
 
-	[#if activity.nurtureDripCampaign?has_content]
+	[#if (activity.nurtureDripCampaign?has_content)! && (activity.nurutureDripCampaing?boolean)! =! false]
 		[#assign digital_marketing_fields = digital_marketing_fields + [{"title": "Nurture or drip campaign?", "value": activity.nurtureDripCampaign?boolean?string("Yes", "No")!"-"}] /]
 	[/#if]
 
-	[#if activity.manySeries?has_content]
-		[#assign digital_marketing_fields = digital_marketing_fields + [{"title": "How many in series?", "value": (activity.manySeries)!"-"}] /]
+	[#if (activity.manySeries?has_content)! && (activity.manySeries?boolean)! != false]
+		[#assign digital_marketing_fields = digital_marketing_fields + [{"title": "How many in series", "value": activity.manySeries}] /]
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.html:42]:
```suggestion
		[#assign digital_marketing_fields = digital_marketing_fields + [{"title": "How many in series", "value": (activity.manySeries)!}] /]
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.html [hunk 21]
```diff
@@ -26,32 +26,32 @@
...
 
-	[#if activity.assetsLiferayRequired == true]
-		[#assign digital_marketing_fields = digital_marketing_fields + [{"title": "Please describe including specifications and due dates", "value": (activity.assetsLiferayDescription)!"-"}] /]
+	[#if (activity.assetsLiferayRequired)! == true]
+		[#assign digital_marketing_fields = digital_marketing_fields + [{"title": "Please describe including specifications and due dates", "value": (activity.assetsLiferayDescription)!}] /]
 	[/#if]
 
-	[#if activity.usingCIABAssets?has_content]
-		[#assign digital_marketing_fields = digital_marketing_fields + [{"title": "Are you using any CIAB assets?", "value": activity.usingCIABAssets?boolean?string("Yes", "No")!"-"}] /]
+	[#if (activity.usingCIABAssets?has_content)! && (activity.usinfCIABAssets?boolean)! != false]
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.html:53]:
```suggestion
	[#if (activity.usingCIABAssets?has_content)!false]
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.html [hunk 22]
```diff
@@ -26,32 +26,32 @@
...
 	[/#if]
 
-	[#if activity.assetsLiferayRequired?has_content]
+	[#if (activity.assetsLiferayRequired?has_content)!]
 		[#assign digital_marketing_fields = digital_marketing_fields + [{"title": "Do you require any assets from Liferay?", "value": activity.assetsLiferayRequired?string("Yes", "No")!"-"}] /]
 	[/#if]
 
-	[#if activity.assetsLiferayRequired == true]
-		[#assign digital_marketing_fields = digital_marketing_fields + [{"title": "Please describe including specifications and due dates", "value": (activity.assetsLiferayDescription)!"-"}] /]
+	[#if (activity.assetsLiferayRequired)! == true]
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.html:49]:
```suggestion
	[#if (activity.assetsLiferayRequired)!false == true]
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.html [hunk 23]
```diff
@@ -124,7 +124,7 @@
 [/#macro]
 
 [#macro leads_table activity]
-	[#assign items = [{"title": "Is a lead list an outcome of this activity?", "value": activity.leadGenerated?string("Yes", "No")!"-" }, { "title": "Target # of Leads", "value": (activity.targetOfLeads)!"-" }, { "title": "Lead Follow Up strategy", "value": (activity.leadFollowUpStrategies)!"-" }, { "title": "Details on Lead Follow Up", "value": (activity.detailsLeadFollowUp)!"-" }] /]
+	[#assign items = [{"title": "Is a lead list an outcome of this activity?", "value": activity.leadGenerated?string("Yes", "No")! }, { "title": "Target # of Leads", "value": (activity.targetOfLeads)!"-" }, { "title": "Lead Follow Up strategy", "value": (activity.leadFollowUpStrategies)!"-" }, { "title": "Details on Lead Follow Up", "value": (activity.detailsLeadFollowUp)!"-" }] /]
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.html:127]:
```suggestion
	[#assign items = [{"title": "Is a lead list an outcome of this activity?", "value": (activity.leadGenerated?string("Yes", "No"))! }, { "title": "Target # of Leads", "value": (activity.targetOfLeads)!"-" }, { "title": "Lead Follow Up strategy", "value": (activity.leadFollowUpStrategies)!"-" }, { "title": "Details on Lead Follow Up", "value": (activity.detailsLeadFollowUp)!"-" }] /]
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Comment by Manda-vit** [comment]
ci:test:sf

**Comment by Manda-vit** [comment]
ci:test:relevant

**Review by jairmedeiros — APPROVED** [review]

**Comment by jairmedeiros** [comment]
resent https://github.com/brianchandotcom/liferay-portal/pull/142055

**Pull request closed without merging**